.gitignore

```diff
@@ -1,4 +1,4 @@
-/node_modules
+node_modules
```

Comment:
Undo this. We want a .gitignore in the example app for it self.

Reply:
Done